db/migrate/20220317235629_create_testimonials.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTestimonials < ActiveRecord::Migration[6.1]
+  def change
+    create_table :testimonials do |t|
+      t.varchar :name, NULL: false
+      t.varchar :content, NULL: true
+
+      t.timestamps
+      t.deletedAt
```

Comment:
Please check this line, because you are adding the `discarded_at` field in the other migration.